Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -13962,6 +13967,11 @@ def main():  # pragma: no cover
         command = demisto.command()
         LOG(f'Command being called is: {command}')
 
+        # Log the API version
+        if is_debug_mode():
+            demisto.debug('Retrieving the API version')
+            demisto.debug(f'API version: {get_pan_os_version()}')
```

Comment:
```suggestion
            demisto.debug(f'PAN-OS Version (debug-mode): {get_pan_os_version()}')
```
I would put it under the `test-module` because assuming that debug-mode would be test on `test-module` this will be executed before the `test-module`